src/Rule/Each.php

```diff
@@ -37,20 +37,27 @@ protected function validateValue($value, ValidationContext $context = null): Res
             return $result;
         }
 
-        foreach ($value as $item) {
+        foreach ($value as $index => $item) {
             $itemResult = $this->rules->validate($item, $context);
-            if ($itemResult->isValid() === false) {
-                foreach ($itemResult->getErrors() as $error) {
-                    $result->addError(
-                        $this->formatMessage(
-                            $this->message,
-                            [
-                                'error' => $error,
-                                'value' => $item,
-                            ]
-                        )
-                    );
+            if ($itemResult->isValid() === true) {
+                continue;
+            }
+
+            foreach ($itemResult->getErrors() as $key => $error) {
+                if (!is_array($item)) {
+                    $errorKey = $index;
+                    $formatMessage = true;
+                } else {
+                    $errorKey = $index . Result::ERROR_KEY_SEPARATOR . $key;
+                    $formatMessage = false;
```

Comment:
Why not format message?

Reply:
Otherwise you get stuff like that in errors:

```php
'charts.2.dots.1.rgb.2' => "Value must be no greater than 255. 259 given. array given. array given."
```

After this change it's returned as expected:

```php
'charts.2.dots.1.rgb.2' => "Value must be no greater than 255. 259 given."

It's needed for correct handling of `Nested` / `Each` combinations
```

Reply:
:heavy_check_mark:

Reply:
Also was thinking about removing value from error message completely, but it's a regression and less intuitive.